welleng/survey.py

```diff
@@ -49,11 +49,7 @@ def __init__(
         deg=True,
         depth_unit='meters',
         surface_unit='meters',
-        mag_defaults={
-            'b_total': 50_000.,
-            'dip': 70.,
-            'declination': 0.,
-        },
+        mag_defaults=None,
```

Comment:
Initialization of function arguments to mutable objects is not recommended in python. Check "Using mutable objects as default argument values in python" of [this page](https://www.geeksforgeeks.org/default-arguments-in-python/).